lib/onelogin/ruby-saml/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module OneLogin
   module RubySaml
-    VERSION = '1.15.0'
+    VERSION = '1.16.0'
```

Comment:
@johnnyshields  I'm more inclined to move to a 2.X version now that we have dropped support for old Ruby versions.

That way we can use 1.16 and 1.17 for security bugfixes and key features we want to add to the old branch.
We should also update the changelog properly.

Reply:
Done! Changed to 2.0.0